tq/meter.go

```diff
@@ -36,6 +32,10 @@ type Meter struct {
 	fileIndex         map[string]int64 // Maps a file name to its transfer number
 	fileIndexMutex    *sync.Mutex
 	updates           chan *tasklog.Update
+
+	DryRun    bool
+	Logger    *tools.SyncWriter
+	Direction Direction
```

Comment:
Nice catch 👍

I wonder if these kind of problems happen often enough to setup a 32-bit build as we did for Git core: https://github.com/git/git/blob/master/ci/run-linux32-build.sh

Reply:
Thanks!

In my opinion, if you're supporting 32-bit, testing on that platform should be a given. Especially, since you guys are using libraries that depend on memory alignment.